catalog/acm/feat-config-mgmt.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeature
+metadata:
+  name: feature-name # kpt-set: gke-acm-config-mgmt-${cluster-name}
```

Comment:
Are there any name restrictions if the cluster name is too big?

Reply:
I can remove some characters. Not sure what the limit on the field is.

Reply:
We previously ran into some char limits on the GCP side thats why I thought I would call out since this seemed long.

Reply:
Reduced the name for these resourced